docs/pages/distribution/publishing-websites.mdx

```diff
@@ -155,50 +143,50 @@ Here are the formal instructions for deploying to GitHub Pages:
 
 - This is probably already done, but if not then you'll want to run `git init` and set the remote.
 
-  ```
+  ```sh
   $ git init
   Initialized empty Git repository in /path/to/expo-gh-pages/.git/
   ```
 
-2. **Add the GitHub repository as a "remote" in your local git repository**
+2. **Add the GitHub repository as a "remote" in your local git repository**.
 
-   ```
+   ```sh
    $ git remote add origin https://github.com/evanbacon/expo-gh-pages.git
    ```
 
    - This will make it so the `gh-pages` package knows where you want it to deploy your app.
    - It will also make it so git knows where you want it to push your source code (i.e. the commits on your `master` branch).
 
-3. **Install the `gh-pages` package as a "dev-dependency" of the app**
+3. **Install the `gh-pages` package as a `dev-dependency` of the app**.
 
    ```sh
    yarn add -D gh-pages
    ```
 
-4. **Configure your **package.json** for web hosting**
+4. **Configure your **package.json** for web hosting**.
 
-   - At the top level, add a `homepage` property. Set it's value to the string `http://{username on github, without the curly brackets}.github.io/{repo-name}`. For example: If my GitHub name is `evanbacon` and my GitHub repository is `expo-gh-pages`, I'll asign the following:
+   - At the top level, add a `homepage` property. Set its value to the string `http://{username on GitHub, without the curly brackets}.github.io/{repo-name}`. For example: If my GitHub name is `evanbacon` and my GitHub repository is `expo-gh-pages`, I'll asign the following:
 
    ```js
-   /* ... */
+   /* @hide ... */ /* @end */
    "homepage": "http://evanbacon.github.io/expo-gh-pages"
    ```
 
    - In the existing `scripts` property, add a `predeploy` property and a `deploy` property, each having the values shown below:
 
-   ```js
+   ```js package.json
    "scripts": {
-     /* ... */
+    /* @hide ... */ /* @end */
      "deploy": "gh-pages -d web-build",
      "predeploy": "expo export:web"
    }
    ```
 
    > In projects running SDK 45 or lower, use `expo build:web` instead of `expo export:web`. `predeploy` is automatically run before `deploy`.
 
-5. **Generate a _production build_ of your app, and deploy it to GitHub Pages.** (2 minutes)
+5. **Generate a _production build_ of your app, and deploy it to GitHub Pages**.
 
-   ```
+   ```sh
    $ yarn deploy
    ```
```

Comment:
We can use `Terminal` in here, but this can be done in separate PR. 🙂

Reply:
I've fixed this one to use `Terminal` component but there are other places on this page where it can be used. Will create a separate PR for that :D